packages/core/types/src/types/core/common/module.ts

```diff
@@ -1,11 +1,15 @@
+import type { PropertyPath } from 'lodash';
 import type { Common, Schema } from '..';
 import type { Strapi } from '../../..';
 
 export interface Module {
   bootstrap: ({ strapi }: { strapi: Strapi }) => void | Promise<void>;
   destroy: ({ strapi }: { strapi: Strapi }) => void | Promise<void>;
   register: ({ strapi }: { strapi: Strapi }) => void | Promise<void>;
-  config<TDefault = unknown, TResult = unknown>(path: string, defaultValue?: TDefault): TResult;
+  config<TDefault = unknown, TResult = unknown>(
```

Comment:
this is valid but the actual code is not going to work if it receives an array packages/core/core/src/domain/module/index.ts